recipe/patches/0001_Fix_xmlErrorPtr.patch

```diff
@@ -0,0 +1,58 @@
+From a13d70254ef1c0788f584aeb5ec25392fe9f8497 Mon Sep 17 00:00:00 2001
+From: Steven <scrass@anaconda.com>
```

Comment:
@SC426 We should give proper attribution with patches. You can get the full patch from https://github.com/PDAL/PDAL/pull/4257.patch. Using the original patch will give us both proper attribution, authorship and commit ID. We can also go one step further and use the patch from the merged commit (https://github.com/PDAL/PDAL/commit/6320e8832e40d9a5f92af619aaaf05169226de1a.patch).

Reply:
Done! Thanks for pointing this out.